crates/compiler/test_syntax/src/bin/minimize.rs

```diff
@@ -46,13 +48,21 @@ fn main() {
         "full" => InputKind::Full,
         "moduledefs" => InputKind::ModuleDefs,
         "header" => InputKind::Header,
+        "pattern" => InputKind::Pattern,
         _ => {
             eprintln!("Invalid input kind: {}", args[index]);
             std::process::exit(1);
         }
     };
+    let input = &args[index + 1];
+    let mut buf = String::new();
+    let text = if input == "-" {
+        std::io::stdin().read_to_string(&mut buf).unwrap();
```

Comment:
Here @Anton-4

Reply:
Hmm, that crate must not be properly marked as for testing because the clippy check ignores tests